notes/health-care-evaluation.vbs

```diff
@@ -2818,6 +2824,9 @@ function write_person_details_in_NOTE()
 		If HEALTH_CARE_MEMBERS(show_hc_detail_const, the_memb) = True Then
 			Call write_variable_in_CASE_NOTE("MEMB " & HEALTH_CARE_MEMBERS(ref_numb_const, the_memb) & " - " & HEALTH_CARE_MEMBERS(full_name_const, the_memb) & " - Processing: " & HEALTH_CARE_MEMBERS(HC_eval_process_const, the_memb))
 			Call write_variable_in_CASE_NOTE("     Status: " & HEALTH_CARE_MEMBERS(hc_eval_status, the_memb))
+			If trim(HEALTH_CARE_MEMBERS(HC_recertification_month, the_memb)) <> "" and trim(HEALTH_CARE_MEMBERS(HC_recertification_year, the_memb)) <> "" Then 
+				Call write_variable_in_CASE_NOTE("     HC Recertification Date: " & HEALTH_CARE_MEMBERS(HC_recertification_month, the_memb) & "/" & HEALTH_CARE_MEMBERS(HC_recertification_year, the_memb))
```

Comment:
I am struggling a little with some of the convention here. 'HC Recert Date' reads a little weird to me. 

I think we could say 'HC Recertification Month:' or we could put the month in the 'Processing' section.

This is the current update:

========================== PERSON DETAILS ==========================
MEMB 01 - MONKEY LUFFY - Processing: Recertification                
     Status: Incomplete - other                                     
     HC Recertification Date: 01/24                                 
     Notes: This is a test case and I just want to see              

I think we can change it to this:
========================== PERSON DETAILS ==========================
MEMB 01 - MONKEY LUFFY - Processing: **01/24** Recertification                
     Status: Incomplete - other                                     
     HC Recertification Date: 01/24   **- Then remove this**                              
     Notes: This is a test case and I just want to see              

There is nothing wrong with it currently, we just typically refer to this as the recert month or MM/YY Revw/Recert.

Reply:
Revised case note to align with the second option above.